R/decontam.R

```diff
@@ -57,6 +57,8 @@
 #' If TRUE, the return value is a \code{data.frame} containing diagnostic information on the contaminant decision.
 #' If FALSE, the return value is a \code{logical} vector containing the binary contaminant classifications.
 #'
+#' @param ... Not used currently
```

Comment:
Why add this?

Reply:
Because the generics need the dots to support other arguments. That is usually done to avoid restricting other developers to arguments you anticipated, when writing the function. (The only solution for another developer would be to add another generic with the same name just to be able to use S4 dispatch for "your" function name.)

For example, to be able to dispatch to `SummarizedExperiment`, I need an argument for the user to select the assay to use. We use `abund_values` or `exprs_values` for such cases, but I can only add these arguments, if the generics allows additional arguments via the dots.

And of course all arguments need to be documented, so thats why they need to be added to the man page.